### DISCUSSION TITLE
Labrador sea freeze over

### DISCUSSION THREAD

**Discussion by adagj** [NorESMhub/noresm3_dev_simulations#43] [Discussions about coupled simulations]
Purpose: How robust is the Lab Sea freeze in NorESM2.5 dev and how do we avoid it?

@matsbn @rosiealice @mvertens 

Some relevant issues / discussions from NCAR:
- https://github.com/NCAR/amwg_dev/issues/188

**Comment by matsbn** [comment]
When discussing the Lab Sea freeze-over with Gustavo Marques, he mentioned two things that was useful for CESM to avoid the freeze-over:
- Distributing the runoff in a narrower coastal region. Traditionally, both CESM and NorESM has distributed runoff in a region close to the coast (typically a radius of 300 km for 1 deg ocean resolution), instead of dumping the runoff in a single coastal grid cell. We have done this mainly to avoid negative salinity, since we convert freshwater flux to a virtual salt flux. By reducing the distribution radius, the idea is that less runoff enters convection locations in the Lab Sea open ocean.
- Get a more realistic initial snow pack to avoid a surge of runoff resulting from unrealistically thick snow pack in initial conditions. I believe Jens looked at runoff into the Lab Sea region of one the latest simulation. As I remember, he found that runoff was indeed large, but unclear if this was an initial surge or a more persistent high runoff flux.

**Comment by adagj** [comment]
The most recent simulation (CLM without FATES, but with the new surface data set with corrected dust), the Arctic is much warmer and the western/coastal part of Lab Sea is much warmer: https://ns2345k.web.sigma2.no/datalake/diagnostics/ADF/plots/n1850.ne30_tn14.hybrid_clmbgc.202401004_5_15_vs_n1850.ne30_tn14.hybrid.20240829_5_15/website/html_img/plot_page_TS_ANN_LatLon_Mean.html

And compared to the CMIP6 NorESM2-MM version, the new simulation is way colder south of Iceland but warmer in the Arctic: https://ns2345k.web.sigma2.no/datalake/diagnostics/ADF/plots/n1850.ne30_tn14.hybrid_clmbgc.202401004_5_15_vs_N1850frc2_f09_tn14_20191001_1204_1214/website/html_img/plot_page_TS_ANN_LatLon_Mean.html

I'm currently running a simulation with FATES. I have only ran 14 years so far, but this simulation also seems warmer in Lab Sea (at least so far): https://ns2345k.web.sigma2.no/datalake/diagnostics/noresm/adagj/n1850.ne30_tn14.hybrid_fatessp.202401007/BLOM_DIAG/yrs5to14-n1850.ne30_tn14.hybrid.20240829/set3/set3_ann_templvl0_2models.png

But running with FATES gives a colder NH and a warmer SH, compared to CLM : https://ns2345k.web.sigma2.no/datalake/diagnostics/noresm/adagj/n1850.ne30_tn14.hybrid_fatessp.202401007/BLOM_DIAG/yrs5to14-n1850.ne30_tn14.hybrid_clmbgc.202401004/set3/set3_ann_templvl0_2models.png

**Comment by oyvindseland** [comment]
Interesting results. At least AOD is much better this time.
Cloud forcing is too low, which is not critical by itself since the global average can be tuned. A more problematic feature is that the stratocumulus regions are almost gone. In addition the Pacific ITCZ is replaced by a large tropical area of convection. This tendency was seen also in the earlier simulations but more pronounced now.
